.github/workflows/build_and_test_library.yml

```diff
@@ -5,9 +5,8 @@ on:
   push:
     tags:
       - "*"
-    paths:
-      - "ansys-grantami-serverapi-openapi/**"
-      - ".github/workflows/build_and_test_library.yml"
+    branches:
+      - main
```

Comment:
This means, I think? That it will run when you update the yaml, it will then run again once the generation workflow completes?

Reply:
ah right, I could not see why we'd limit that for tags, but the filters apply to all types of pushes. That's somewhat annoying, as on the other hand, we want this run on develop regardless of the modified files

Reply:
Oh right. Yes. Github removed the context. Yes this should be fine.